clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/screenshare/services/ScreenshareMessageReceiver.as

```diff
@@ -0,0 +1,9 @@
+package org.bigbluebutton.lib.screenshare.services
+{
+	public class ScreenshareMessageReceiver

```

Comment:
Duplicate class. This one can be removed.

Reply:
Deleted.